Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance.yml

```diff
@@ -255,7 +255,7 @@ tasks:
       asmattacksurfacerulepriority:
         simple: ${ASM.AttackSurfaceRule.priority}
       asmattacksurfaceruleremediationguidance:
-        simple: ${ASM.AttackSurfaceRule.remediation_guidance}
+        simple: ${ASM.AttackSurfaceRule.remediation_guidance}${RemediationGuidance}
```

Comment:
It's safer to use the `SetIfEmpty` transformer for such cases, please use it instead

Reply:
[96eac55](https://github.com/demisto/content/pull/37244/commits/96eac559ac4b5aa1268bd54ec319624595c7bfd8)